helm_deploy/laa-crime-applications-adaptor/templates/external-ingress.yaml

```diff
@@ -30,32 +34,23 @@ spec:
     {{- end }}
   {{- end }}
   rules:
-    {{- range .Values.ingress.hosts }}
-    - host: {{ .host | quote }}
+    {{- if .Values.ingress.externalHost }}
+    - host: {{ $.Values.ingress.externalHost.host | quote }}
       http:
         paths:
-          {{- range .paths }}
-          - path: {{ . }}
+          - path: {{ $.Values.ingress.externalHost.path }}
             pathType: "Prefix"
             backend:
               service:
                 name: {{ $fullName }}
                 port:
                   number: {{ $svcPort }}
           - path: {{ $.Values.actuator.health.path }}
-            pathType: "Prefix"
+            pathType: "Exact"
             backend:
               service:
-                name: {{ $fullName }}
-                port:
-                  number: {{ $.Values.actuator.port }}
-          - path: {{ $.Values.actuator.metrics.path }}
-            pathType: "Prefix"
-            backend:
-              service:
-                name: {{ $fullName }}
+                name: {{ $fullName }}-external
```

Comment:
I think this one needs to be just `{{ $fullName }}` as everything target the same backend K8s service.

Reply:
Yes, I was getting desperate (will remove) as I can't seem to get it to work. It keeps complaining that the endpoint is alright defined in the ingress.
<img width="1309" alt="Screenshot 2024-02-15 at 09 14 31" src="https://github.com/ministryofjustice/laa-crime-applications-adaptor/assets/12012077/fc14c456-8c88-46f4-ad2e-471b21471b52">

Reply:
CDA that does have it working have different hosts for internal and external ingresses - I wonder if that is why it works for them based off of this. However, the paths are different for the ingresses...